### ISSUE TITLE
delete is not working

### ISSUE THREAD

**Issue by apegrave** [marckrenn/framer-Firebase#9]
hi, why when im trying to delete some node

```
a = new Layer
a.onTap ->
	firebase.delete("/appMoney/test_minus/-LBSQUlsxfwun0UQVa")
```
it doesn't do anything. Any idea why it's not working. Other methods are working (firebase.get is working)

**Comment by marckrenn** [comment]
Steve fixed this problem in #10

Could you confirm this? Thanks.

**Comment by apegrave** [comment]
now it's working! thx!
